pkg/storage/unified/sql/sqltemplate/dialect_sqlite.go

```diff
@@ -16,3 +16,8 @@ type sqlite struct {
 	argPlaceholderFunc
 	name
 }
+
+func (sqlite) CurrentEpoch() string {
+	// Alternative approaches like `unixepoch('subsecond') * 1000000` returns millisecond precision.
+	return "CAST((julianday('now') - 2440587.5) * 86400000000.0 AS INTEGER)"
```

Comment:
that is wild! seems like it's the blessed way to do it though

Reply:
well TIL what julian days are lol

Reply:
🤯  wow

Reply:
Absolutely glad that we will just have to call this function instead of copy-pasting this formula anywhere lol. Amazing work finding this out!